src/libraries/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -1051,6 +1051,8 @@ private bool TryAddInternal(TKey key, int? nullableHashcode, TValue value, bool
         private static void ThrowKeyNotFoundException(TKey key) =>
             throw new KeyNotFoundException(SR.Format(SR.Arg_KeyNotFoundWithKey, key.ToString()));
 
+        public IEqualityComparer<TKey> Comparer => _comparer ?? _defaultComparer;
```

Comment:
Nit: please add an xml comment to help seed the documentation for the new member.